drv_priv.h

```diff
@@ -54,6 +54,7 @@ struct driver {
 	const struct backend *backend;
 	void *priv;
 	void *buffer_table;
+	int is_sriov;
```

Comment:
suggest to use one common type, like enum CIV_GPU_TYPE {
ONE_GPU
TWO_GPU_SRIOV
TWO_GPU_IGPU_DGPU
THREE_GPU_SRIOV_DGPU
}